BIBLIOGRAPHY.md

```diff
@@ -48,8 +48,8 @@ source code and documentation.
 * Referenced from:
   - [README.md](README.md)
   - [examples/bring_your_own_fips202/mldsa_native/mldsa_native.h](examples/bring_your_own_fips202/mldsa_native/mldsa_native.h)
-  - [examples/bring_your_own_fips202/mldsa_native/src/common.h](examples/bring_your_own_fips202/mldsa_native/src/common.h)
   - [examples/bring_your_own_fips202/mldsa_native/src/config.h](examples/bring_your_own_fips202/mldsa_native/src/config.h)
+  - [examples/bring_your_own_fips202/mldsa_native/src/ct.h](examples/bring_your_own_fips202/mldsa_native/src/ct.h)
```

Comment:
nit: typo s in the commit message of 89428ff3ea310504ff7241905b7d9da8372595ae

Reply:
Fixed the typos.